crates/telio-dns/src/zone.rs

```diff
@@ -44,21 +51,27 @@ impl AuthoritativeZone {
         }
         let zone_name = Name::from_str(name)?;
         let zone = InMemoryAuthority::empty(zone_name.clone(), ZoneType::Primary, false);
-
+        let ttl_value_signed: i32 = match ttl_value.0.try_into() {
+            Ok(ttl_value) => ttl_value,
+            Err(_) => {
+                telio_log_warn!("TTL value could not be converted from u32 to i32 without data loss, so using default value");
```

Comment:
I think this warning would be more useful if it contained the actual value which will be used instead of only the word "default".